.travis.yml

```diff
@@ -34,13 +34,13 @@ jobs:
         - ./scripts/test-setup.sh
         - export PATH=$PATH:/usr/bin/geckodriver
       script:
-        - python3 codalab_service.py test --version ${TRAVIS_BRANCH} default
-          # Make sure restarting worker doesn't cause any issues (ie in serialization/deserialization)
-        - docker restart codalab_worker_1 && python3 codalab_service.py test run --version ${TRAVIS_BRANCH}
+        - python3 test_runner.py --version ${TRAVIS_BRANCH} default
+        # Make sure restarting worker doesn't cause any issues (ie in serialization/deserialization)
+        - docker restart codalab_worker_1 && python3 test_runner.py --version ${TRAVIS_BRANCH} run
         - docker stop codalab_worker_1
-        - python3 codalab_service.py start --services worker --version ${TRAVIS_BRANCH} --shared-file-system
-        - python3 codalab_service.py test --version ${TRAVIS_BRANCH} --shared-file-system basic run read write kill resources netcat netcurl workers
-        - python3 tests/ui/ui_tester.py --headless
+        - python3 codalab_service.py start --services init worker --version ${TRAVIS_BRANCH} --shared-file-system
+        # Also run the frontend tests in this iteration of testing
```

Comment:
I created a module `frontend` that `test_runner` recognizes and runs the front end tests.